src/gui/vsDevice.cpp

```diff
@@ -474,13 +472,13 @@ QString VsDevice::randomString(int stringLength)
         "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789");
 
     if (!seeded) {
-        qsrand(QTime::currentTime().msec());
+        m_randomizer.seed((QTime::currentTime().msec()));
```

Comment:
Thanks for this. I wasn't sure how to update this one.

Reply:
No problem. There are more secure ways to do this without needing to seed things, but didn't know if you needed predictable results from the generator.